fasm-plugin/tests/Makefile

```diff
@@ -1,6 +1 @@
-TESTS = dummy
-
-all: $(TESTS)
-
-dummy:
-	@echo $@ PASS
+include $(shell pwd)/../../Makefile_test.common
```

Comment:
This needs a test?  Or it just doesn't have a test?

Reply:
It does not have a test. We should open an issue to make one. From what I recall that was the first plugin and we somehow didn't add a testsuite.

Reply:
Please create an issue, but I didn't believe this required blocking the merge, etc.